ingestors/kafka/src/main/scala/hydra/kafka/programs/KeyAndValueSchemaV2Validator.scala

```diff
@@ -196,6 +207,9 @@ class KeyAndValueSchemaV2Validator[F[_]: Sync] private (schemaRegistry: SchemaRe
 }
 
 object KeyAndValueSchemaV2Validator {
+
+  final val DEFAULT_LOOPHOLE_CUTOFF_DATE_KEY = "DEFAULT_LOOPHOLE_CUTOFF_DATE_IN_YYYYMMDD"
+  final val DEFAULT_LOOPHOLE_CUTOFF_DATE_DEFAULT_VALUE = "20230609"
```

Comment:
This date will be updated based on the input from @Jodi.

Reply:
TODO: After the review is done, we will update this date as the date of pushing this PR out to staging/prod as mentioned [here in slack](https://pluralsight.slack.com/archives/C04RUEL448H/p1686587039988709?thread_ts=1686583332.187229&cid=C04RUEL448H). Thank you, @znickle24.

Reply:
Move it to app configs.

Reply:
Done.